ts/features/itwallet/machine/eid/selectors.ts

```diff
@@ -71,3 +71,6 @@ export const selectUpgradeFailedCredentials = (snapshot: MachineSnapshot) =>
         >
     )
   );
+
+export const selectCredentialTypeOption = (snapshot: MachineSnapshot) =>
```

Comment:
Do we need it as an `Option`? We only check if it's set or not, could an `undefined` check be enough?